car.capnp

```diff
@@ -312,7 +312,7 @@ struct CarState {
 
   # deprecated
   errorsDEPRECATED @0 :List(CarEvent.EventName);
-  brakeLights @19 :Bool;
+  brakeLightsDEPRECATED @19 :Bool;
```

Comment:
Let's preserve this for now.
```suggestion
  brakeLights @19 :Bool;
```

Reply:
This is intentional for the refactor, it's already taken care of on the sync

Reply:
Ah gotcha, didn't see you updated in the other places. too.